aggregator/src/test/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJobTest.java

```diff
@@ -37,6 +38,7 @@ void init() {
     }
 
     @Test
+    @Disabled // TODO Test disabled because it blocked a prod deployment.
```

Comment:
@bennetelli This is the failing test I mentionned this morning on Slack.

Reply:
I fixed this in #474 by adding

    @BeforeEach
    void setUp() {
        helpers.purgeAggregations();
        helpers.purgeEmailSubscriptions();
    }

    @AfterEach
    void tearDown() {
        helpers.purgeAggregations();
        helpers.purgeEmailSubscriptions();
    }